modules/conversion-center/widgets/link-in-bio.php

```diff
@@ -24,7 +24,7 @@
  * @since 3.23.0
  */
 class Link_In_Bio extends Widget_Base {
-	const TAB_ADVANCED = 'advanced-tab-custom';
+	const TAB_ADVANCED = 'advanced-tab-link-in-bio';
```

Comment:
Is `link-in-bio` just the name of this one widget? If so, can we make this name something more generic like `links-manager` or something?

Reply:
I'm changing it to `advanced-tab-links`, I needed a way to differentiate it from the core `advanced-tab`

Reply:
done